pkg/ebpf/tracer_windows.go

```diff
@@ -19,10 +18,14 @@ const (
 
 var (
 	expvarEndpoints map[string]*expvar.Map
-	expvarTypes     = []string{"state", "driver_total_flow_stats", "driver_flow_handle_stats", "total_flows", "open_flows", "closed_flows", "more_data_errors"}
 )
 
 func init() {
+	expvarTypes := []string{"state"}
+	for _, n := range network.DriverExpvarNames {
+		expvarTypes = append(expvarTypes, string(n))
+	}
+
```

Comment:
Why is this changing ?

Reply:
See https://github.com/DataDog/datadog-agent/pull/6709#discussion_r518856286